handlers.go

```diff
@@ -390,11 +419,22 @@ func (b *BouncerHandler) shouldAttribute(reqParams *BouncerParams) bool {
 		}
 	}
 
+	// Check if the request is coming from RTAMO, and if so, only attribute
+	// if there is a referer header from https://www.mozilla.org/
+	// https://github.com/mozilla-services/go-bouncer/issues/347
+	if fromRTAMO(reqParams.AttributionCode) {
```

Comment:
Presumably in go, reqParams are implicitly urldecoded?

Reply:
@muffinresearch `reqParams` are created in this function which takes the URL encoded parameters and sets (decodes) the values accordingly: https://github.com/mozilla-services/go-bouncer/blob/347/params.go#L21-L31

Inside of the attribution code parameter, there is a base64 encoded value that once decoded can be further URL decoded which happens in the `fromRTAMO` function.

Here's an example showing the different steps to decode the URL and get to the section where we can tell it's from RTAMO: https://play.golang.org/p/5FQf29j0P-C